src/EnergyPlus/GroundHeatExchangers.cc

```diff
@@ -115,11 +114,6 @@ namespace EnergyPlus::GroundHeatExchangers {
 // Xiong, Z., D.E. Fisher, J.D. Spitler. 2015. 'Development and Validation of a Slinky
 //   Ground Heat Exchanger.' Applied Energy. Vol 114, 57-69.
 
-// Using/Aliasing
-using namespace DataLoopNode;
-
-using namespace GroundTemperatureManager;
-
 // MODULE PARAMETER DEFINITIONS
 constexpr Real64 hrsPerMonth(730.0); // Number of hours in month
 constexpr Real64 maxTSinHr(60);      // Max number of time step in a hour
```

Comment:
We don't have this defined anywhere else?

Reply:
I can't find anything